schema/me/consignments/submission.js

```diff
@@ -63,6 +63,24 @@ export const SubmissionCategoryAggregation = new GraphQLEnumType({
   },
 })
 
+export const SubmissionStateAggregation = new GraphQLEnumType({
+  name: "SubmissionStateAggregation",
+  values: {
+    DRAFT: {
+      value: "draft",
+    },
+    SUBMITTED: {
+      value: "submitted",
+    },
+    APPROVED: {
```

Comment:
Just noting that even though `APPROVED` and `REJECTED` are valid states, clients should not be _setting_ the submission to those states. We will also be adding more internal states and I'm not sure if metaphysics needs to keep supporting them.

Reply:
Maybe it's better to just outright do DRAFT/SUBMITTED for now then